src/main/java/org/openjdk/backports/report/csv/PushesCSVReport.java

```diff
@@ -45,9 +45,11 @@ protected void doGenerate(PrintStream out) {
 
         for (Issue i : model.byTime()) {
             String pushUser = Accessors.getPushUser(i);
-            out.printf("\"%s\", \"%s\", \"%s\", \"%s\"%n",
+            out.printf("\"%s\", \"%s\", \"%s\", \"%s\", \"%s\",  \"%s\"%n",
```

Comment:
Removing the spaces between the fields allows the file to be opened properly in Excel
`out.printf("\"%s\",\"%s\",\"%s\",\"%s\",\"%s\",\"%s\"%n",`